frontend/src/metabase/query_builder/utils.unit.spec.ts

```diff
@@ -243,14 +244,6 @@ describe("isNavigationAllowed", () => {
       ).toBe(true);
     });
 
-    it("allows to run a model", () => {
```

Comment:
This test was invalid (copy-paste remnants) - it's for a model, but the `describe` is for a question